Sources/Conformance/main.swift

```diff
@@ -120,6 +120,8 @@ func singleTest() throws -> Bool {
 }
 
 Google_Protobuf_Any.register(messageType: Conformance_TestAllTypes.self)
+Google_Protobuf_Any.register(messageType: ProtobufTestMessages_Proto3_TestAllTypes.self)
```

Comment:
The registration per se only needs to be done once at program start; after that, the registry itself is read-only.  We could serialize access, but that seems overkill right now.

Note that we are not simply unpacking the Any field in this case.   Rather, we're decoding a protobuf binary message (which may potentially be quite complex with Any fields buried at any point), then re-encoding as JSON.  As a side-effect of transcoding, Any fields buried within the message must be unpacked and repacked.  This in turn requires access to the type.  Unlike Obj-C, Swift has no mechanism for dynamically locating a type by name at run-time.

To avoid the registry, we could either:
 * Pass in a collection of types to the serializer (similar to how we handle extensions during decoding)
 * Require the user to manually inspect every Any field and unpack/rewrite it (which converts an Any field holding a protobuf to an Any field holding a message -- the latter can then be serialized to any supported format)